### ISSUE TITLE
✨ First-Timers Only Issues! 🌈 Make & Request Your Issue!

### ISSUE THREAD

**Issue by TildaDares** [publiclab/plots2#11105]
Hi everyone, we at Public Lab are very welcoming to contributors especially open source newcomers and people from underrepresented groups. Contributing to Open-Source can be overwhelming especially for first-timers and that's why we created this issue to make it easier for you to contribute.

We have issues that we call FTOs(First Timers Only) that are meant for first-timers only. These issues give you a step-by-step guide on how to contribute and what files need to be changed. You don't have to know any programming language to work on these issues. These issues are designed to help you learn our workflow here at Public Lab.

### How Do I Get an FTO Issue?

Please leave a comment below and we'll add your name to the list. If you're an Outreachy or GSOC applicant, please tell us and we'll give you priority. 

We'll tag you when there's an issue for you to work and you have **24 hours** to claim it. If you're stuck, please feel free to reach out and ask for help in our [community chat](https://gitter.im/publiclab/publiclab). 

### How To Claim an FTO?

Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.

And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.

If the issue was reserved for you, please claim it **within 24 hours**.

### I've worked on an FTO, what next?

Congratulations 🎉, you've made your first Public Lab Contribution. We always encourage contributors to create FTOs for others or lookout for issues with the `bug` /`help wanted` tags.

If you'd like to create an FTO for someone else, you can look through the repo or [publiclab.org](https://publiclab.org/) for issues that can be solved in a few lines. When you've found an issue for a first-timer to work on, click the **New Issue** button and use the issue template named `First Timers Only issue`. 

If you have any trouble, feel free to reach out to us on our [community chat](https://gitter.im/publiclab/publiclab) and we'll do our best to help you resolve it. Thank you 🎈

**Comment by PlusUltra404** [comment]
Hello First timer here. I would love o work on an issue.

**Comment by TildaDares** [comment]
Welcome to Public Lab @PlusUltra404. I'll add your name to the list.

**Comment by willmelton21** [comment]
Hi! I'm a first timer and I'd love to work on something!

**Comment by TildaDares** [comment]
Hi @PlusUltra404, there’s an issue for you in https://github.com/publiclab/plots2/issues/10937 if you’re interested.

**Comment by danjkim21** [comment]
Hello there! First timer, happy to help however I can.

**Comment by PlusUltra404** [comment]
> Hi @PlusUltra404, there’s an issue for you in #10937 if you’re interested.

Thank you,

**Comment by Dustpie** [comment]
Hello Tilda, thank you for your message!
I would like to work on an issue, this is my first time.

**Comment by FoleyDom** [comment]
Hey Tilda! First timer here, that would love to help and learn.

**Comment by ronran1** [comment]
Hi Tilda, thanks for doing this. I would love to work on a first-timers-only issue. I have beginner-level knowledge of HTML, CSS, JS, Node, and ReactJS.

**Comment by djmiller1008** [comment]
Hello Tilda! First-Timer here looking to help and learn!

**Comment by TildaDares** [comment]
Hi @willmelton21, there’s an issue reserved for you in https://github.com/publiclab/plots2/issues/11109 for 24 hours!

**Comment by TildaDares** [comment]
Hi @danjkim21, there’s an issue reserved for you in https://github.com/publiclab/infragram/issues/407 for 24 hours!

**Comment by nawed2611** [comment]
Hey! @TildaDares I'm looking to get started with some first-time issues!

**Comment by danjkim21** [comment]
> Hi @danjkim21, there’s an issue reserved for you in [publiclab/infragram#407](https://github.com/publiclab/infragram/issues/407) for 24 hours!

Thank you Tilda. WIll get started right away.

**Comment by cmcgee0308** [comment]
Hi Tilda! I've contributed to an issue and was wondering how I would create an issue for others to work on?

**Comment by TildaDares** [comment]
Hi @cmcgee0308, you can start by identifying a simple issue that a first-timer could fix. You can take a look at other FTOs for inspiration https://github.com/publiclab/plots2/labels/first-timers-only

 When you've found an issue for a first-timer to work on, click the **New Issue button** and use the issue template named **First Timers Only issue**.

**Comment by YorhaBlue** [comment]
Hi! I'm new here, but this is really cool - would love to contribute to a First-Timers issue. I'm beginner/intermediate-ish with HTML, CSS, and JS.

**Comment by fedechaves** [comment]
Hello Tilda! I would love to work on a FTO issue. thanks in advance!

**Comment by TildaDares** [comment]
Hi @Dustpie, there’s an issue reserved for you in #11115 for 24 hours!!

**Comment by Dustpie** [comment]
Thank you Tilda!

**Comment by TildaDares** [comment]
Hi @FoleyDom, there’s an issue reserved for you in https://github.com/publiclab/mapknitter/issues/1683 for 24 hours!!

**Comment by TildaDares** [comment]
Hi @ronran1, there’s an issue reserved for you in https://github.com/publiclab/plots2/issues/11120 for 24 hours!!

**Comment by TildaDares** [comment]
Hi @djmiller1008, there’s an issue reserved for you in https://github.com/publiclab/community-toolbox/issues/562 for 24 hours!!

**Comment by TildaDares** [comment]
Hi @nawed2611, there’s an issue reserved for you in https://github.com/publiclab/community-toolbox/issues/563 for 24 hours!!

**Comment by nawed2611** [comment]
Thankss!

**Comment by TildaDares** [comment]
Hi @YorhaBlue, there's an issue reserved for you in https://github.com/publiclab/community-toolbox/issues/568 for 24 hours!

**Comment by YorhaBlue** [comment]
Thank you @TildaDares !

**Comment by istarlet** [comment]
Hi @TildaDares , first timer here. I would love to help to help out.

**Comment by TildaDares** [comment]
Hi @fedechaves, there's an issue reserved for you in https://github.com/publiclab/plots2/issues/10987 for 24 hours!!

**Comment by TildaDares** [comment]
Hi @istarlet, there's an issue reserved for you in https://github.com/publiclab/mapknitter/issues/1662. Thanks!!

**Comment by istarlet** [comment]
Thank you, @TildaDares ! I accept.

**Comment by z-prince** [comment]
Would love to bed added here to! Primary skills are Ruby/Rails and PostgreSQL.

**Comment by christinecannon** [comment]
Hello, I would like to be added!

**Comment by christinecannon** [comment]
Thank you @TildaDares I will get started working right away!

**Comment by gunesnt** [comment]
Hi @TildaDares I would like to work on an FTO issue. Thanks

**Comment by gunesnt** [comment]
Hi @TildaDares I would like to work on an FTO issue. Thanks!

**Comment by ornevirardi** [comment]
Hello!! First timer here! I would love to contribute 😄

**Comment by nikhilnarla** [comment]
Hi @TildaDares I would love to contribute to FTO issues!

**Comment by matthewjholmes** [comment]
Hi there! Would love to contribute.

**Comment by htinz** [comment]
Hi! @TildaDares, First Timer here, I am excited to contribute to FTO issues.

**Comment by jenniferhalloran** [comment]
Hi there! First timer here. Would love to contribute :)

**Comment by gitofbabai** [comment]
Hi @TildaDares, Can you please assign me a FTO issue. I would love to contribute.

**Comment by pokacyb** [comment]
Hey @TildaDares if there's something I can help with please share :)

**Comment by RafiyaZafar** [comment]
> Hi @RafiyaZafar, there's an issue reserved for you in #11208 for 24 hours!
Thanks Tilda.

**Comment by pokacyb** [comment]
> Hi @pokacyb, there's an issue reserved for you in #11207 for 24 hours!

Thank you

**Comment by shourya2002-geek** [comment]
Hello, first-timer here, I would like to contribute! I have good knowledge of MERN Stack.

**Comment by udyanojha** [comment]
Hi everyone! open-source
This is Udyan, I'm new to open source and would love to contribute to the open-source community. Please let me know what I can work on.

**Comment by Abdullahqasim75** [comment]
i am first timer i want to work

**Comment by lblk-coder** [comment]
Hello @TildaDares, I'm first timer, and'd be happy to help !

**Comment by XD69Z** [comment]
HI @TildaDares, I want to contribute to open source, and I am a first timer. Anything that I can help out with?

**Comment by leighbriody** [comment]
Hi , I would love a first issue to work on !

**Comment by bad-atnames0** [comment]
Hi @TildaDares  I would like a first issue to work on

**Comment by noleni** [comment]
Hi @TildaDares, I'd love to contribute :)

**Comment by abhiramshukla** [comment]
Hi, first timer here. Would love to contribute :)

**Comment by jindalj** [comment]
Hi, @TildaDares i would love to contribute in the first issue

**Comment by thexjoker71** [comment]
Hello there! First timer, happy to help however I can.

**Comment by Nishchay1571999** [comment]
Hi @TildaDares I would love to contribute in the first issue.

**Comment by nzaih1999** [comment]
hello id love a task to work on

**Comment by swayamterode** [comment]
Hello I'm beginner to Open Source I haven't contributed anything so far and would love to get my PR merged!

**Comment by Nishchay1571999** [comment]
Hi @TildaDares I wasnt available for the past few days can have another issue for the beginners once again ?

**Comment by Nishchay1571999** [comment]
Hi @TildaDares #11310 code for review.

**Comment by Harsh3363** [comment]
Hii @TildaDares  I want to get started with open source contribution

**Comment by vaishnavi4444** [comment]
Hi @TildaDares I would like to contribute and I'm new to open source, could you add me to FTO list please?

**Comment by hsadia538** [comment]
Hi @TildaDares can you add my name to the FTO list as well and assign an issue? Thanks

**Comment by Manaswee03** [comment]
Hi, @TildaDares I would like to contribute for the first issue

**Comment by Harsh3363** [comment]
> Hi @Harsh3363, there's an issue reserved for you in https://github.com/publiclab/plots2/issues/11319 for 24 hours!

Thanks for informing ☺

**Comment by nzaih1999** [comment]
Hello @TildaDares my laptop got spoilt and I didn't see my issue..any chance that I can be assigned another issue?

**Comment by slayeh17** [comment]
Hello @TildaDares  I am very exited to make my first contribution. 
Can I get an issue to work on 😊

**Comment by PeculiarE** [comment]
Hi @nzaih1999, there's an issue reserved for you in https://github.com/publiclab/mapknitter/issues/1775 for 24 hours!

**Comment by hyy170190** [comment]
Hi. This is my first time to contribute 😄 Can I have a FTO issue?

**Comment by zexinbeh** [comment]
Hello, I am a beginner and I want to work on a FTO issue. Can you grant my request? :D

**Comment by xinzhe822** [comment]
Hi. This is my first time contributing to an open-source project. Can I have an FTO issue😀?

**Comment by april-EXO** [comment]
Hello~ I'm new to contributing to an open-source project. Can I have an FTO issue?

**Comment by prak30** [comment]
hey new to open source would love to contribute to something

**Comment by opelolo** [comment]
Hello there 😄 I'm new to opensource and would love to contribute. Thank you

**Comment by Aadil-Shaik** [comment]
Hello @TildaDares  could you help me look for an issue
Thankyou

**Comment by slayeh17** [comment]
> Hi @slayeh17, there's an issue reserved for you in #11333 for 24 hours!

Thank You so much @TildaDares

**Comment by hyy170190** [comment]
> Hi @hyy170190, there's an issue reserved for you in #11334 for 24 hours!

Thank you.

**Comment by zexinbeh** [comment]
Hi. This is my first time to contribute for this project 😄 Can I have a FTO issue?

**Comment by zexinbeh** [comment]
> Hi @zexinbeh, there's an issue reserved for you in #11340 for 24 hours!

Thank you @TildaDares

**Comment by gogoiakash2311** [comment]
hello, I'd love to work on an issue.

**Comment by iem-saad** [comment]
Hi @TildaDares, I am a ROR developer having experience of 2 years. But I am new to open source can you please assign me something to get started?

Thanks.

**Comment by april-EXO** [comment]
> Hi @april-EXO, there's an issue reserved for you in https://github.com/publiclab/plots2/issues/11342 for 24 hours!

Thank you very much <3

**Comment by prak30** [comment]
> Hi @prak30, there's an issue reserved for you in #11343 for 24 hours!

Thanks tilda, starting to work on it

**Comment by opelolo** [comment]
> Hi @opelolo, there's an issue reserved for you in #11344 for 24 hours!

Thank you. I'm on it now

**Comment by zadilkhwaja** [comment]
Hi! I'm a first timer and I'd love to work on something!

**Comment by Beckeruni** [comment]
Hey, first timer here.
    Would love to get something to work on :)

**Comment by zadilkhwaja** [comment]
> Hi @zadilkhwaja, there’s an issue reserved for you in #11353 for 24 hours.

Accepted👍🏻

**Comment by TildaDares** [comment]
Hi @Beckeruni, there's an issue reserved for you in https://github.com/publiclab/plots2/issues/11357 for 24 hours!

**Comment by nickmendezFlatiron** [comment]
@nickmendezFlatiron  , would like to contribute for the first time

**Comment by Akshath11** [comment]
Hi there , Its my first time contributing . Would love to get something to work on.

**Comment by TildaDares** [comment]
Hi @nickmendezFlatiron, there’s an issue reserved for you in #11354 for 24 hours!

**Comment by zadilkhwaja** [comment]
Can i get 1 more?

**Comment by TildaDares** [comment]
Hi @zadilkhwaja, contributors can only work on one first-timers only issue. But you can look at the issues tab and pick an issue that interests you to work on. Thank you!

**Comment by paulaxisabel** [comment]
Hello @TildaDares! I'm new to open source and would love to contribute. I'd be happy to assist in any way I can. Thank you very much!

**Comment by caciulla** [comment]
HI! I'm a first-timer learning git workflow and I would love to have an issue to work on!

**Comment by TildaDares** [comment]
Hi @Akshath11, there's an issue reserved for you in #11376 for 24 hours!

**Comment by caciulla** [comment]
Hi Tilda, I think there was a mix-up. I'm actually @caciulla,
not @Akshath11.

**Comment by TildaDares** [comment]
Hi @caciulla, we have a list at the top that is arranged according to who requested for an issue first. 

Be rest assured that we’ll look for an issue for you.

**Comment by Sumer312** [comment]
Hi, first timer here, would love to work on something.

**Comment by TildaDares** [comment]
Hi @paulaxisabel, there's an issue reserved for you in #11388 for 24 hours!

**Comment by TildaDares** [comment]
Hi @caciulla, there's an issue reserved for you in #11389 for 24 hours!

**Comment by TildaDares** [comment]
Hi @Sumer312, there's an issue reserved for you in #11390 for 24 hours!

**Comment by Wafa9** [comment]
Hello Tilda,
I would like to work on an issue, this is my first time.

**Comment by ojninja16** [comment]
hi @TildaDares ,
I would like to contribute in any way I can ...

**Comment by Mshehu5** [comment]
Hello Tilda,
I would like to make a contribution

**Comment by cheehwatang** [comment]
Hi @TildaDares , I am a first-timer and would like to work on an issue. Thank you!

**Comment by lanefoxwood** [comment]
Heya, any FTO issue I can try? It's my first time and I'd love to contribute.
Thanks!

**Comment by ryuusama09** [comment]
hi ,I am  a first timer . Can i claim this issue ? @TildaDares

**Comment by pbnj1** [comment]
Hello,

I would like to participate and try a FTO please.  Thank you!

**Comment by jcod3s** [comment]
Hello! I would like to participate! Could I please be added to the FTO list? Thank you!

**Comment by melwhite1** [comment]
Hello! I am a first-timer! I would love to work on an issue.

**Comment by TildaDares** [comment]
Hi @Wafa9, there's an issue reserved for you in #11406 for 24 hours!

**Comment by TildaDares** [comment]
Hi @Mshehu5, there's an issue reserved for you in #11407 for 24 hours!

**Comment by TildaDares** [comment]
Hi @ryuusama09, there's an issue reserved for you in #11408 for 24 hours!

**Comment by giftade** [comment]
Hello, I'd like to work on an issue

**Comment by OlawaleYaqub** [comment]
Hello, I have beginner knowledge in html, CSS, and Js and would love to work on an issue

**Comment by Sayak-singha** [comment]
Hello Tilda, 
I am a beginner and would like to work on an issue

**Comment by cesswairimu** [comment]
Hi @pbnj1 you can claim and work on this one https://github.com/publiclab/plots2/issues/10306

**Comment by cesswairimu** [comment]
Hi @jcod3s, I have reserved this for you.

**Comment by jcod3s** [comment]
> Hi @jcod3s, I have reserved this for you #11416.

Great! Thank you - I'll get started!

**Comment by cesswairimu** [comment]
Hi @melwhite1, here is one https://github.com/publiclab/plots2/issues/11417

**Comment by cesswairimu** [comment]
hey @giftade, this one is reserved for you https://github.com/publiclab/plots2/issues/11418. Thanks

**Comment by abhishek0chauhan** [comment]
Hello, I am a First timer in open source contribution. I am looking to work on an issue.

**Comment by cesswairimu** [comment]
hi @OlawaleYaqub you can work on this -  https://github.com/publiclab/plots2/issues/11424

**Comment by TildaDares** [comment]
Hi @Sayak-singha, there's an issue reserved for you in #11418.

**Comment by Vinayakkamble07** [comment]
Hello First timer here. I would love to work on an issue.

**Comment by priyda** [comment]
Hi, I am new to the open source community and I want to work on an issue.

**Comment by megative** [comment]
Hey, I'm a new into opensource so would be great to start with something

**Comment by Code-Captain7** [comment]
I would like to work on an FTO .

**Comment by vamshidhar-kasulabada** [comment]
Hello First Timer here. I'd like to work on an issue

**Comment by TildaDares** [comment]
Hi @abhishek0chauhan, there's an issue reserved for you in #11429 for 24 hours!

**Comment by TildaDares** [comment]
Hi @Vinayakkamble07, there's an issue reserved for you in https://github.com/publiclab/plots2/issues/11430 for 24 hours!

**Comment by TildaDares** [comment]
Hi @megative, there's an issue reserved for you in https://github.com/publiclab/plots2/issues/11431 for 24 hours!

**Comment by TildaDares** [comment]
Hi @Code-Captain7, there's an issue reserved for you in https://github.com/publiclab/plots2/issues/11433 for 24 hours!

**Comment by TildaDares** [comment]
Hi @vamshidhar-kasulabada, there's an issue reserved for you in https://github.com/publiclab/plots2/issues/11434 for 24 hours!

**Comment by vamshidhar-kasulabada** [comment]
Thank u for assigning me @TildaDares . I will start working on it

**Comment by Aman-Jamshed** [comment]
Hey @TildaDares , I'm new here and also an Outreachy applicant.I would love to work on FTO issue. Would you kindly assign some issue to me?
Thanks in advance!

**Comment by yuhwaa** [comment]
Hi @TildaDares ! I'd like to work on a first-timer issue if any opens up! Thank you!

**Comment by Narayan1670** [comment]
Hi @TildaDares, I am new to open-source, Can you assign me any issue ?

**Comment by TildaDares** [comment]
Hi @Aman-Jamshed, there's an issue reserved for you in #11430 for 24 hours!

**Comment by cesar-codes** [comment]
Hi @TildaDares ! I would love to work on an issue if one opens up. Thanks!

**Comment by TildaDares** [comment]
Hi @yuhwaa, there's an issue reserved for you in https://github.com/publiclab/plots2/issues/11445 for 24 hours!

**Comment by TildaDares** [comment]
Hi @Narayan1670, there's an issue reserved for you in https://github.com/publiclab/plots2/issues/11446 for 24 hours!

**Comment by TildaDares** [comment]
Hi @cesar-codes, there's an issue reserved for you in https://github.com/publiclab/plots2/issues/11447 for 24 hours!

**Comment by DevPJ9** [comment]
Hello @TildaDares I would love to work on an issue.

**Comment by cesswairimu** [comment]
Hi @DevPJ9, this one is reserved for you https://github.com/publiclab/plots2/issues/11451

**Comment by tiff-o** [comment]
hey @TildaDares - keen to work on an issue :) TIA

**Comment by AnnaS40** [comment]
Hi @TildaDares! I'm new to open source and would love to work on any issue if one opens up!

**Comment by TildaDares** [comment]
Hi @tiff-o, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1038 for 24 hours!

**Comment by TildaDares** [comment]
Hi @AnnaS40, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1039 for 24 hours!

**Comment by DevPJ9** [comment]
Hi @TildaDares. Looking forward to work on an issue.

**Comment by TildaDares** [comment]
Hi @DevPJ9, contributors are only allowed to work on one first timers issue. You can look for issues with the `help-wanted` or `bug` label to work on. You can also choose to create a first-timer issue for someone else. Thank you!

**Comment by toshitapandey** [comment]
Hey, @TildaDares! Looking forward to work on an issue.

**Comment by TildaDares** [comment]
Hi @toshitapandey, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1042 for 24 hours.

**Comment by kmje** [comment]
Hey @TildaDares! I'm a first timer. I'd like to work on an issue.

**Comment by TildaDares** [comment]
Hi @kmje, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1043 for 24 hours.

**Comment by AndroGenius-codes** [comment]
Hello, Good evening @TildaDares I am a first timer from outreachy and i am happy to help

**Comment by Khadeeejah** [comment]
hi @TildaDares i am a first timer from outreachy and i would love to contribute to this project, kindly assign an issue to me. thank you

**Comment by TildaDares** [comment]
Hi @AndroGenius-codes, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1047 for 24 hours.

**Comment by brendamoreira** [comment]
Hello @TildaDares, I'm a outreachy applicant searching for a first time issue to make a contribution :) thanks

**Comment by MAX-786** [comment]
Hello @TildaDares,i am looking for first time issue to make contribution . Thanks :-)

**Comment by Dezzy12** [comment]
Hello, i am looking fir first time issue to make contribution too

**Comment by mofirojean** [comment]
Hello @TildaDares, I'm a outreachy applicant searching for a first time issue to make a contribution

**Comment by ayrhkhan** [comment]
Hi @TildaDares! I'm a first-timer and I'd love to work on something! Thank you

**Comment by FrancoFrancis** [comment]
> 

hello im an outreachy applicant I would love to get a FTO please thanks

**Comment by NPDebs** [comment]
Hello @TildaDares 👋🏼 
I'm an Outreachy participant and I would love to contribute to this project.

P.S. I'm also a first-timer. 🙂

**Comment by jesutobi** [comment]
Hello @TildaDares my name is Jesutobi , i am an outreachy intern, and front-end developer and would love to contribute to this project

**Comment by leilayesufu** [comment]
Hello @TildaDares my name is Leila , i am an outreachy applicant , and a front-end developer with knowledge in HTML,CSS ,bootstrap, javascript and Jquery and would love to be assigned an issue

**Comment by ifeawoyemi** [comment]
Hi @TildaDares my name is Ifeoluwa I am an Outreachy applicant, I would love to contribute to this project and assigned an issue also this is my first time to contribute to open a source project

**Comment by Lovelyfin00** [comment]
HI Tilda, I'm Loveth and an outreachy applicant. I would love to be assigned a first timers issue. Thank you😀

**Comment by ijayhub** [comment]
hi Tilda, I'm Ijeoma an outreachy applicant. first time , I will like to contribute. Thank you

**Comment by dayopius** [comment]
Hi Tilda, I'm a first timer & an Outreachy applicant, I'd love to be assigned an issue please. Thank you :)

**Comment by MAX-786** [comment]
Hello there, I would like to work on a issue can u assign me one . Thanks😊

**Comment by TildaDares** [comment]
Hi @Khadeeejah, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1066 for 24 hours.

**Comment by belfarz** [comment]
Hello tilda I'm an outreachy applicant and also a first timer i would love to work on a FTO issue

**Comment by TildaDares** [comment]
Hi @brendamoreira, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1067 for 24 hours.

**Comment by TildaDares** [comment]
Hi @MAX-786, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1068 for 24 hours.

**Comment by TildaDares** [comment]
Hi @mofirojean, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1069 for 24 hours.

**Comment by sahilbansal17** [comment]
Hi @TildaDares, I would like to work on an issue.

**Comment by TildaDares** [comment]
Hi @ayrhkhan, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1070 for 24 hours.

**Comment by TildaDares** [comment]
Hi @FrancoFrancis, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1071 for 24 hours.

**Comment by TildaDares** [comment]
Hi @NPDebs, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1072 for 24 hours.

**Comment by NPDebs** [comment]
> Hi @NPDebs, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1072 for 24 hours.

Thank you so much! 🙌🏼

**Comment by mofirojean** [comment]
> Hi @mofirojean, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1069 for 24 hours.

Thank you ☺️

**Comment by OluwatuyiAbiola** [comment]
Hello, @TildaDares  I'm an Outreachy participant and I would love to contribute to this project. I'm also a first-timer.

**Comment by adetyaz** [comment]
Hello, I'm from outreachy and I'm looking for my first contribution.

**Comment by klaus139** [comment]
Hello, first Timer here, I would like to work on an issue

**Comment by leilayesufu** [comment]
Hello @TildaDares my name is Leila , i am an outreachy applicant 
I’d like to be assigned an FTO issue

**Comment by klaus139** [comment]
Hi @TildaDares I would like to be assigned an issue. Thanks

**Comment by TildaDares** [comment]
Hi everyone, we appreciate your enthusiasm to make a contribution. There’s a list in the issue description above with the names of people who have requested for an issue. If your name is there or you’ve left a comment already, please refrain from leaving another comments so that we can avoid duplicated names. Thank you and we look forward to your contributions!

**Comment by chemistry-notfun** [comment]
Hello @TildaDares, I'm an outreachy applicant and a first-timer, I would like to work on a issue. Thank you!

**Comment by codeinvoke7** [comment]
Hello @TildaDares, I'm Elijah an outreachy applicant please I'll like to work on an issue

**Comment by Ibom99** [comment]
Hey @TildaDares I am an Outreachy applicant. I'll like to kindly be assigned to an FTO issue, Thank You!

**Comment by titiodus** [comment]
Hello @TildaDares I am an Outreachy applicant. I'll like to be kindly assigned to an issue - one about accessibility or documentation. Thank you

**Comment by titiodus** [comment]
Hello @TildaDares. I am an Outreachy applicant and would love to contribute to any issues with Project 1. Thank you

**Comment by THEDARKMAN007** [comment]
hello @TildaDares. my name is Samad and I am an outreachy applicant id like to work on FTO issues. thank you very much

**Comment by abdelquodr** [comment]
Hello, @TildaDares. I am an Outreachy applicant, i'll like to be assigned an FTO issue

**Comment by albinazs** [comment]
Hello, @TildaDares! I'm an Outreachy applicant. Looking forward to make valuable contributions to Public Lab!

**Comment by ChiomaNwosisi** [comment]
Hello @TildaDares i am an applicant, i'll like to be assigned an FTO issue

**Comment by mutmainaho** [comment]
Hello @TildaDares, I would like to be assigned an FTO issue

**Comment by obueto** [comment]
Hello @TildaDares  ,I  would like to be assigned an FTO issue

**Comment by oluwatunbi** [comment]
Hello @TildaDares, I have not been assigned any task to work on. Thank you.

**Comment by Tammy-Ajoko** [comment]
Hello I'm Tammy, an outreachy applicant. I would like to apply for an FTO 😊.

**Comment by leumasxxx** [comment]
hello I'm Samuel an outreachy applicant and id lov to be assigned a FTO

**Comment by Sally-N** [comment]
hello @TildaDares I'm Sally and I'd like to be assigned an FTO

**Comment by haszankauna** [comment]
Hello , I'm Kauna an outreachy applicant and I'll love to be assigned a task.

**Comment by TaladeogoA** [comment]
Hello @TildaDares. I am an outreachy applicant, an I would like to be assigned to an issue. Thank you.

**Comment by Munnachienyi** [comment]
hello, @TildaDares. I am an outreachy applicant and also new to open source. would love to contribute to the first timers issue

**Comment by TomisinAjayi** [comment]
Hi @TildaDares my name is Tomisin and I am an outreachy applicant. 
I’d like to work on an FTO issue. Thank you!

**Comment by codeinvoke7** [comment]
@TildaDares you listed my name but have not assigned any issue to me

**Comment by Jennifer-tech** [comment]
hello @TildaDares , I'm an outreachy applicant and also new to open source. I would like to be assigned an issue. Thanks

**Comment by Ranecodes** [comment]
Hello @TildaDares. I am an outreachy applicant and I'll like to contribute and work on a FTO issue. Thank you

**Comment by Jennifer-tech** [comment]
hello @TildaDares , I'm an outreachy applicant and also new to open source. I would like to be assigned an issue. Thanks

**Comment by meggiddy** [comment]
Hello, @TildaDares, I'm an Outreachy Applicant and I'd like to get an issue assigned to me. Thank You!

**Comment by xiaodaiweiwei** [comment]
Hello @TildaDares , I'm Astrid Liu and I'm an Outreachy Applicant. I'm new to the open source but I have the passion to work on some issuers so I would like to apply for an FTO issue. Thank you!

**Comment by kabuur** [comment]
Hello I'm Ahmed an outreachy aplicant also I'm First timer open source contribution

**Comment by tiffieangel** [comment]
Hello @TildaDares , I'm Tifany-Angel, an Outreachy applicant. I'd like an FTO issue to work on.

**Comment by sainamercy** [comment]
Hi @TildaDares, I'm an outreachy applicant. This will be my first time contributing to your code base. I would appreciate a First Timer Issue to work on. Thanks

**Comment by Thesaleem** [comment]
Hi @TildaDares. I'm an outreachy applicant, and I would like to be assigned an FTO issue

**Comment by THEDARKMAN007** [comment]
hello, @TildaDares. I am an outreachy applicant, I would love to be assigned an issue. thank you.

**Comment by Julietadeboye** [comment]
Hello @TildaDares, I'd love to be assigned an issue. Thanks

**Comment by Tammy-Ajoko** [comment]
@TildaDares don't know if I did it right the first time by not tagging you, but I'll like to apply for an FTO

**Comment by xiaodaiweiwei** [comment]
Hi Ahmed,&nbsp;
Nice to meet you! We could help each other and get over the first time haha.
Astrid


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "publiclab/plots2"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月10日(星期一) 下午3:55
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [publiclab/plots2] ✨ First-Timers Only Issues! 🌈 Make &amp; Request Your Issue! (Issue #11105)





 
Hello I'm Ahmed an outreachy aplicant also I'm First timer open source contribution

**Comment by Straw4yu** [comment]
Hi @TildaDares I'll love to be assigned an issue. I'm new here. Thank you.

**Comment by Bhaybhiisparks** [comment]
Hi @TildaDares  I am an outreachy applicant. I'd love to be assigned an FTO issue

**Comment by Jennifer-tech** [comment]
hello @TildaDares , I'm an outreachy applicant and also new to open source. I would like to be assigned an issue. Thanks

**Comment by Josephsanya** [comment]
Hello @TildaDares Please, I'm an Outreachy applicant, and would like to be assigned an FTO issue. Thanks.

**Comment by IdayatSanni** [comment]
Hello @TildaDares, I'm an outreachy applicant and would like to be assigned an issue. Thank you

**Comment by Tkwenan** [comment]
Hi @TildaDares I'm an Outreachy applicant and I'm waiting to be assigned an issue. Thanks.

**Comment by Divine-peace** [comment]
Hello @TildaDares , thanks for the email. I'm an applicant waiting to be assigned an issue. Please, help me.

**Comment by Divine-peace** [comment]
Hello @TildaDares, thanks for the email. Please, I need an FTO issue to work on.

**Comment by Favour-Cay** [comment]
Hii @TildaDares, im a first timer and ill like if you can please assign me an issue to work on

**Comment by iLynette** [comment]
Hi @TildaDares, I am also just getting started and would appreciate it if I could be assigned an issue(s) to work on. Thanks.

**Comment by banjo-tech** [comment]
Hi @TildaDares, 
I'm an Outreachy applicant. Kindly help out on how I can get assigned to an issue. Many thanks.

**Comment by mimi-tech** [comment]
Hi @TildaDares, Hope you are doing well. Please can I get assigned to a task? Thank you.

**Comment by TildaDares** [comment]
Hi @jesutobi, @leilayesufu, @ifeawoyemi, @Lovelyfin00, @ijayhub, @dayopius, @belfarz, @sahilbansal17, @OluwatuyiAbiola, @adetyaz, @klaus139, @chemistry-notfun, @codeinvoke7, @Ibom99, @titiodus, @THEDARKMAN007, @abdelquodr, @albinazs, @ChiomaNwosisi, @mutmainaho, @obueto, @oluwatunbi, @Tammy-Ajoko, @leumasxxx, @haszankauna, @TaladeogoA, @Munnachienyi, @TomisinAjayi, @codeinvoke7, @Jennifer-tech, @Ranecodes, @meggiddy, @xiaodaiweiwei, @kabuur, @tiffieangel, @sainamercy, @RealRichi3, @Thesaleem, @THEDARKMAN007

Please take a look at this issue https://github.com/publiclab/Leaflet.DistortableImage/issues/1086 and work on the files assigned to you. Leave a comment on the issue to let us know you're working on the file assigned to you. If you have any questions, you can ask them in the issue. Thanks!

**Comment by segun-codes** [comment]
Hi @TildaDares, I think one step is missing in the section "Standard Installation" (on https://github.com/publiclab/plots2/blob/master/README.md), I couldn't get the plot2 app running on gitpod without the step. Is it okay, I take that up as an issue and fix same?

**Comment by Mootconn** [comment]
Hi @TildaDares I would like to be assigned an issue. Thanks!

**Comment by TildaDares** [comment]
Hi @segun-codes, can you open an issue for that with a brief description of the change you want to make? Thanks!

**Comment by segun-codes** [comment]
> Hi @segun-codes, can you open an issue for that with a brief description of the change you want to make? Thanks!



> Hi @segun-codes, can you open an issue for that with a brief description of the change you want to make? Thanks!

Okay, thanks.

**Comment by segun-codes** [comment]
@TildaDares I have now opened an issue as advised. Here is it _Update ReadMe.md file to include a missing installation step #11466_

**Comment by ayeolakenny** [comment]
Hi @TildaDares can I get assigned to a task? Thank you.

**Comment by Achele** [comment]
Hello @TildaDares , I'm  an Outreachy applicant. I'd like an FTO issue to work on

**Comment by onujoy14** [comment]
Hello @TildaDares, I'll like to be assigned to an issue. I'm an Outreachy applicant

**Comment by olamideazeez** [comment]
Hello @TildaDares, I'll like to contribute as a first timer, i am an Outreachy applicant

**Comment by victory-lydia** [comment]
Hello, @TildaDares, I'll like to contribute as a first timer, I am an outreachy applicant.

**Comment by Ceejaycodez** [comment]
Hi @TildaDares. I'll be leaving the comment here waiting to be assigned an issue. Thanks

**Comment by TildaDares** [comment]
Hi @ayeolakenny, looks like you have worked on a `first-timers-only` issue before. You can try helping others who are just making their first contribution or find another issue without the `first-timers-only` label. Thank you!

**Comment by Sally-N** [comment]
Hi @TildaDares , I'm from outreachy, I'd like to contribute as a first timer

**Comment by sharon-kathambi** [comment]
Hello @TildaDares , I'm an Outreachy applicant, I'd like to contribute as a first Timer

**Comment by victorvictoria-maker** [comment]
Good day @TildaDares, I'm one of Outreachy's applicants. I would like to contribute as a first-timer on this project. Thank you.

**Comment by Kemmie001** [comment]
Hello @TildaDares, I am an outreach applicant, I would like to contribute as a first-timer on this project. Thank you.

**Comment by Jolah1** [comment]
Hi @TildaDares, I am an outreachy applicant, I would like to contribute as a first-timer on this project. 
Thank you.

**Comment by M0Hassan** [comment]
Hi @TildaDares, I am an Outreachy applicant and a first-timer. I would like to be assigned an issue. Thank you.

**Comment by victorvictoria-maker** [comment]
Hello @TildaDares. I am a first-timer. I would be glad if a task is assigned to me.
Victoria Victor, Outreachy applicant.

**Comment by julianavmgiusti** [comment]
@TildaDares Hi! Would like to contribute with an issue as outreachy applicant.

**Comment by fonkwe** [comment]
Hi @TildaDares,
I'm an Outreachy applicant. Kindly help out on how I can get assigned to an issue.

**Comment by tolubishops0** [comment]
Hello Tilda,
I am an outreachy applicant, kindly help me out to get started

**Comment by RealRichi3** [comment]
Hello @TildaDares  how can i add tags to an already created issue, I created an issue #11464  for a missing installation step in the README

**Comment by fonkwe** [comment]
Hello @TildaDares , I just made my first PR: https://github.com/publiclab/Leaflet.DistortableImage/pull/1110

**Comment by THEDARKMAN007** [comment]
Hi @TildaDares i just made my first pull request at https://github.com/publiclab/Leaflet.DistortableImage/pull/1099

**Comment by maryojo** [comment]
Hi @TildaDares , I'm an Outreachy applicant and I would like to contribute to an FTO issue. Thank you!

**Comment by ForJessicaSake** [comment]
Hi @TildaDares I'm an Outreachy applicant and I'd like to be assigned to an issue please

**Comment by Emenyi95** [comment]
Hello @TildaDares i'll like to contribute to the first timer projects. Thank you.

**Comment by ChiomaNwosisi** [comment]
Hello @TildaDares , I just made my first PR: https://github.com/publiclab/Leaflet.DistortableImage/pull/1109

**Comment by Ceejaycodez** [comment]
Hi @TildaDares. I am an outreachy applicant. Are there any unclaimed FTO's currently that I can contribute to?

**Comment by JulianaSau** [comment]
Hi @TildaDares , I am an outreachy applicant and would like to work on an issue

**Comment by Simplecodez** [comment]
Hi, I am Emmanuel Elendu, an Outreachy applicant.

**Comment by ChiomaNwosisi** [comment]
Sorry @TildaDares I'm just confused about the whole process please could
you explain to me what I should do now that I have finished my first issue,
am I suppose to work on more or help other people with their first issue.

—

**Comment by alexanderchosen** [comment]
Good day @TildaDares I am an outreachy applicant. Please is there any FTO issue to contribute to? I'd really appreciate it

**Comment by thenextxchapter** [comment]
Hello @TildaDares I am an Outreachy applicant and I'd like to work on an issue. Thanks

**Comment by damli40** [comment]
Hey, I’m a first timer and I’ll like to get an issue

**Comment by liliyao2022** [comment]
Dear @Tilda, I'm Lili Yao, an Outreachy applicant, I'd like to contribute as a First Timer. Thanks!

**Comment by Frank-Cisca** [comment]
Hello @TildaDares
I'm an outreachy applicant and I'd love to contribute/work on an issue as a first timer. Kindly help me out on this. Thanks.

**Comment by Frank-Cisca** [comment]
Hello @TildaDares 
I am an Outreachy applicant and I'd love to contribute/work on an issue as a first timer. Kindly help me out with this. Thanks!

**Comment by Alwoch** [comment]
Hello @TildaDares , outreachy applicant here, I'd like to try out a first timer's issue please

**Comment by sainamercy** [comment]
Hi, @Julietadeboye issue #474 has been reserved for you. Please check it out. 
If you bump into challenges, feel free to reach out.

**Comment by oryxwhite** [comment]
Hello @TildaDares, I'm an Outreachy applicant and would love to contribute.

**Comment by Julietadeboye** [comment]
> Hi, @Julietadeboye issue [publiclab/infragram#474](https://github.com/publiclab/infragram/issues/474) has been reserved for you. Please check it out. If you bump into challenges, feel free to reach out.

I will like to claim this issue

**Comment by Chelsofia** [comment]
Hi @TildaDares I'm an outreachy applicant and looking forward to getting an issue.

**Comment by Emenyi95** [comment]
Hello @TildaDares . I will like to work on an FTO. Please can one be assigned to me?

**Comment by davidarebuwa** [comment]
Hi @TildaDares! Fellow outreachy applicant looking to work on some FTO issues

**Comment by AminMusah** [comment]
@TildaDares Hi! Would like to contribute with an issue as outreachy applicant.

**Comment by JeffyDave** [comment]
Hi @TildaDares, please I am looking for a first timer issue to contribute as an outreachy applicant. Please can I be assigned to one?

**Comment by Chinazaekpere-Ike** [comment]
Heyy @TildaDares 
I'm a first time and I'd like to contribute to an issue as an outreachy applicant

**Comment by 7malikk** [comment]
Hello, @TildaDares, I'm an Outreachy applicant and I'd like to contribute to public lab in any capacity. I'm a front-end web developer. I see you have a lot of applicants like me looking for FTOs, how can I make this process easier for you?

**Comment by alexanderchosen** [comment]
Good day @TildaDares I am Alexander, an outreachy applicant from Nigeria. I am an entry full-stack developer who is passionate about open-source contributions. Please is there any issue I can work on?
Thanks

**Comment by rsmith-github** [comment]
Hi! I would be happy to work on an FTO issue.

**Comment by ayeolakenny** [comment]
Hello @TildaDares i would like to wok an FTO issue, could you add my name on the list, thanks

**Comment by laskelly** [comment]
Hi @TildaDares 
I'll be glad if you add my name to the list for FTO. Thank you

**Comment by IvyJeptoo** [comment]
Hello Tilda my name is Ivy and I am an Outreachy First Timer Applicant, kindly add my name to the the FTO list.
Thank you.

**Comment by TildaDares** [comment]
> Hello @TildaDares i would like to wok an FTO issue, could you add my name on the list, thanks

Hi @ayeolakenny, looks like you have worked on a first-timers-only issue before. You can try helping others who are just making their first contribution or find another issue without the first-timers-only label. Thank you!

**Comment by TildaDares** [comment]
> Hi everyone, we appreciate your enthusiasm to make a contribution. There’s a list in the issue description above with the names of people who have requested for an issue. If your name is there or you’ve left a comment already, please refrain from leaving another comment so that we can avoid duplicated names. Thank you and we look forward to your contributions!

Leaving one comment per contributor helps me track what names have not been added to the list. If you leave more than one comment, there's a chance that I'll miss names that have not been added. Spamming this issue with multiple comments does not mean you'll get an issue faster. As long as you've left a comment, we'll add your name to the list. Thanks everyone!

**Comment by Jem256** [comment]
Hey @TildaDares I am an outreachy applicant and I am looking for a FTO issue to contribute to

**Comment by Emenyi95** [comment]
Hi @TildaDares . I have requested for an issue since yesterday and none has been assigned to me.

**Comment by Onyi-X** [comment]
Hi, I'm an Outreachy applicant. Looking foward to be assigned an FTO

**Comment by Teslimsama** [comment]
Hey Ian outreachy applicant FTO I would like to assigned a task

**Comment by RoleyJes** [comment]
Hello, I'm Jessica, an Outreachy applicant. I'd love to be added to the list

**Comment by hugochooqu** [comment]
Hello @TildaDares i would like to wok an FTO issue, could you add my name on the list, thanks

**Comment by hillaryCyn** [comment]
Hello @TildaDares i would like to wok an FTO issue, could you add my name on the list, thanks

**Comment by SylviaKaks** [comment]
hello, Am an applicant of Outreachy. I would like to get an FTO issue and contribute in any way i can

**Comment by Julietadeboye** [comment]
Hi, @Josephsanya issue [#11473](https://github.com/publiclab/plots2/issues/11473) has been reserved for you. Please check it out.

**Comment by Josephsanya** [comment]
Please kindly review my PR at #11476 @TildaDares 
Thanks.

**Comment by Josephsanya** [comment]
Thank you @Julietadeboye 
I have submitted a PR which fixes the issue

**Comment by Natacha-Danbadan** [comment]
Hi everyone, I'm Natacha, an Outreachy Applicant and I would loved to be assigned an FTO @TildaDares. Thanks.

**Comment by aweSAM-XS** [comment]
Hi @TildaDares, 
I'm a first timer and an Outreachy applicant. Kindly add me to the list. Thank you.

**Comment by Dhei-vid** [comment]
Hey my name is David and I am an outreachy applicant, and I am new to contributing to open source. Kindy add me to the list. Thank you.

**Comment by hillaryCyn** [comment]
hello, Am an applicant of Outreachy. I am new to open source issue and i would like  contribute in any way i can. @TildaDares

**Comment by segun-codes** [comment]
Hi @TildaDares my PR 11468 is still waiting on your review, could you kindly attend to it (here https://github.com/publiclab/plots2/pull/11468). Many thanks...

**Comment by 7malikk** [comment]
Hello, @TildaDares, and @cesswairimu, kindly review my PR at [#11480](https://github.com/publiclab/plots2/pull/11480)
Thank you as i anticipate your feedback

**Comment by MARVEBUKA** [comment]
hi @TildaDares . first timer from outreachy here. I would love to work on a FTO issue. thanks in advance.

**Comment by TildaDares** [comment]
Hi @IdayatSanni, there’s an issue reserved for you in #11446.

**Comment by AryaCollection** [comment]
Hi @TildaDares, I am a first timer from Outreachy. I would like to be assigned a FTO issue.

**Comment by emmyclef** [comment]
Hello @TildaDares , I am an html, css and javscript developer from outreach looking to work on a FTO issue

**Comment by Emmanuel-Lud** [comment]
Hi @TildaDares , I'm a first timer from outreachy. I would love to work on a FTO issue.

**Comment by boluwatifee** [comment]
Hello, I'm a first timer and an Outreachy applicant. I would like to be assigned a FTO issue @TildaDares

**Comment by TildaDares** [comment]
Hi @Tkwenan, there's an issue reserved for you in https://github.com/publiclab/plots2/issues/11486 for 24 hours. Thanks!

**Comment by sanmay321** [comment]
Hello, I'm a first-timer and I'd love to work on something! please add me to the list.

**Comment by 7malikk** [comment]
@Divine-peace an FTO has been reserved for you [here](https://github.com/publiclab/plots2/issues/11487), kindly claim it within the next 24hours

**Comment by lizwe-mac** [comment]
Hi @TildaDares 
I'm a first timer looking for an FTO. I'm an entry level software developer with ample knowledge in the MERN stack

**Comment by lizwe-mac** [comment]
Hi @TildaDares 
I'm a first timer looking for a FTO. I'm an entry level full stack developer with ample skills in the MERN stack.

**Comment by AdityaVerma32** [comment]
Hello @TildaDares , I would love to work on FTO issue. Plz Add me the list.

**Comment by TechieTee** [comment]
Hello @TildaDares, my name is Modupe, and I am an outreachy applicant, I will like to make my first contribution to open source. Kindly add me to the list to be able to make my first. Thank you and weldone.

**Comment by LynetteMwe** [comment]
Hello, @TildaDares.
I'm a first-timer and would be glad to work on an FTO issue. Kindly add me to the list.

**Comment by faithybest** [comment]
Hello @TildaDares, I‘m an Outreachy applicant. I am interested in working on FTO issue.

**Comment by Frank-Cisca** [comment]
Hello @TildaDares 
I am an Outreachy applicant. Please I left a comment since on an FTO issue to work on and I haven't gotten any reply yet. Kindly help me out with this. Thanks!

**Comment by EricaP451** [comment]
Hi @TildaDares, I am an  Outreachy applicant and it is my first time contributing to open source. I am excited to be assigned a FTO issue and make my contribution. Thank you.

**Comment by vareenrose** [comment]
Hi @TildaDares I am an Outreachy applicant and I would love to get an FTO

**Comment by IvyJeptoo** [comment]
Hello @TildaDares I have seen a tick on my name meaning I have been allocated an FTO, How do I get to see the allocation since it has not been tagged under my comment.

**Comment by cesswairimu** [comment]
hi @IvyJeptoo, here is the issue https://github.com/publiclab/plots2/issues/11492

**Comment by onireke** [comment]
Hello, @TildaDares I would like to work on the FTO issue. Please, kindly add me to the list. 
Thanks a lot!

**Comment by onireke** [comment]
Hello, @TildaDares I would like to work on the FTO issue. Please, kindly add me to the list. 
Thanks a lot!

**Comment by Tonathe3rd** [comment]
Hi! I'm a first timer and I'd love to work on something!
Please, Kindly add me to the list

**Comment by Tishly** [comment]
Hello @TildaDares, I'm an Outreachy applicant. I would appreciate being assigned an FTO issue to work on. Thank you.

**Comment by Tonathe3rd** [comment]
> Hi! I'm a first timer and I'd love to work on something! Please, Kindly add me to the list

@TildaDares  Kindly help assign me to an FTO issue to work on. Thank very much

**Comment by Augusta2** [comment]
Hello @TildaDares Can you please assign me to an issue?

**Comment by charmilahruthika** [comment]
hey @TildaDares can you please assign me to a FTO issue ?

**Comment by Wafa9** [comment]
Hello, I'm a first timer and an Outreachy applicant. I would like to be assigned a FTO issue @TildaDares

**Comment by TildaDares** [comment]
Hi @VictoriaAjala, FTOs are reserved for first-timers to this project. You can work on other issues without the `first-timers-only` label or choose to create issues for other people to work on. Thank you!

**Comment by safa-ismail** [comment]
Hello @TildaDares I am an outreachy intern and first timer to this project. I have mid level experience in Html, Css and JS. I would like to be assigned on a FTO issue

**Comment by Achele** [comment]
Hello @TildaDares I have seen a tick on my name meaning I have been allocated an FTO, How do I get to see the allocation since it has not been tagged under my comment.

**Comment by lizwe-mac** [comment]
Hi @TildaDares I'm looking to work FTOs

**Comment by TildaDares** [comment]
Hi @tolubishops0, there is an issue reserved for you here https://github.com/publiclab/Leaflet.DistortableImage/issues/1146.

**Comment by onireke** [comment]
Hi @TildaDares please I am yet to be assigned a FTO the only thing 
I saw was that my comment was deleted. Kindly assist me. Thank you!

**Comment by TildaDares** [comment]
Hi @onireke, sorry about that. I deleted a duplicate of your comment so that there’s only a single copy of the comment.

**Comment by Chelsofia** [comment]
Hi @TildaDares  I still have no issue. And deadline is almost here. I'm an outreachy applicant.

**Comment by RoleyJes** [comment]
@TildaDares please, can I be assigned an issue?

**Comment by samailamalima** [comment]
Hi @TildaDares @jywarren  @cesswairimu, my name is Samaila Anthony Malima I'm an outreachy intern, and I would love to contribute to this project please how can I start?

**Comment by TEGS-OMAT** [comment]
Hi, I'm Tegs, an outreachy intern. I would love to contribute to this project. please assign me to an FTO

**Comment by tolubishops0** [comment]
@TildaDares GoodMorning. I see a name on my name but I do not know where to find the task assigned to me pls??
how do I go about it?

**Comment by tolubishops0** [comment]
> Hi @tolubishops0, there is an issue reserved for you here [publiclab/Leaflet.DistortableImage#1146](https://github.com/publiclab/Leaflet.DistortableImage/issues/1146).

Thank you @TildaDares I'm on it

**Comment by TildaDares** [comment]
Hi @Favour-Cay, there’s an issue reserved for you here #1160.

**Comment by TildaDares** [comment]
Hi @onujoy14, there's an issue reserved for you in https://github.com/publiclab/plots2/issues/11490 for 24 hours!

**Comment by TildaDares** [comment]
Hi @Ceejaycodez, there's an issue reserved for you in #11429 for 24 hours!

**Comment by yukiyuli** [comment]
@TildaDares do you have any issues that I can work on?

**Comment by charmilahruthika** [comment]
@toshitapandey do you have any issues to assign?

**Comment by TildaDares** [comment]
Hi @Sally-N, there’s an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1183 for 24 hours.

**Comment by TildaDares** [comment]
Hi @sharon-kathambi, there’s an issue reserved for you in #1185 for 24 hours!

**Comment by TildaDares** [comment]
Hi @olamideazeez, there’s an issue reserved for you here https://github.com/publiclab/Leaflet.DistortableImage/issues/1218.

**Comment by Brujitari** [comment]
Hello is there an issue I could work on? @TildaDares 🥰

**Comment by zannyb** [comment]
Hello, I am an outreachy applicant, and I would love to contribute to this project during my internship.
Which FTO issue could I work on?

**Comment by zannyb** [comment]
Hello, I am an outreachy applicant, and I would love to contribute to this project during my internship.
Which FTO issue could I start with?

**Comment by JeffyDave** [comment]
hello @TildaDares I would love to help you in creating and/or allocating FTO issues. I see there's a lot of work load on that, I'd love to lighten the work load.

**Comment by cynthiarichard007** [comment]
Hello @TildaDares I'm an Outreachy applicant, and i would like to contribute to any project available, this is my first time and I'm happy to be here.

**Comment by Abigail-cloud** [comment]
Hello @TildaDares , My name is Oluwadare Abigail and a first-timer to Open-source contributions. I am an Outreachy applicant and I would love to contribute on the project. 
Thank you for your kind consideration.

**Comment by victoria-parker** [comment]
Hello! Is there any issue I could work on?? Thank you :)

**Comment by FrancoFrancis** [comment]
> Hello! Is there any issue I could work on?? Thank you :)

hey @victoria-parker  yes you can look through the list of already open issues to find an issue you are interested in and work on creating a pull request  or you can request for a first timers only(fto) issue to work on.

**Comment by TildaDares** [comment]
Hi @Kemmie001, there’s an issue reserved for you here https://github.com/publiclab/Leaflet.DistortableImage/issues/1235.

**Comment by TildaDares** [comment]
Hi @Onyi-X, there's an issue reserved for you here https://github.com/publiclab/Leaflet.DistortableImage/issues/1160 for 24 hours!.

**Comment by jssol** [comment]
Hello @TildaDares, I am an Outreachy applicant and I would like to get an FTO issue.

**Comment by Aditya31398** [comment]
Hello @TildaDares, I am a first timer. I would like to work on an issue.

**Comment by SilasEze** [comment]
Hi, I'm Aries, an outreachy intern. I would love to contribute to this project. please assign me to an FTO.

**Comment by actuallyyun** [comment]
Hi, I'm Yun, I would love to get started contributing to open-source projects!

**Comment by TildaDares** [comment]
Hi @lizwe-mac, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1253 for 24 hours!

**Comment by TildaDares** [comment]
Hi @yukiyuli, there's an issue reserved for you in https://github.com/publiclab/Leaflet.DistortableImage/issues/1086 for 24 hours!

**Comment by AhVir** [comment]
Hi @TildaDares, I would love to get myself started with open-source projects. Can I have **an issue** to work on?

**Comment by NadifaHersi** [comment]
@TildaDares , @cesswairimu , @jywarren , after I have fixed an issue can I record the contribution or try to work some other issues and record all contributions at one time?

**Comment by nikhilgeo** [comment]
Hi @TildaDares , I like to contribute to this project, it would be great if you could tag me on a FTO issue when possible.

**Comment by TildaDares** [comment]
Hi @Jolah1, there's an issue reserved for you here https://github.com/publiclab/Leaflet.DistortableImage/issues/1174 for 24 hours.

**Comment by Jolah1** [comment]
@TildaDares i'm working on it now
thank you.

**Comment by victoria-parker** [comment]
Hello @TildaDares! I would love to start contributing, could you assign me a FTO issue? Thank you a lot!

**Comment by harshhh11** [comment]
Hi Tilda, First-Timer here would love to work on a issue

**Comment by TildaDares** [comment]
Hi @victoria-parker, there's an issue reserved for you in https://github.com/publiclab/plots2/issues/11485.

**Comment by SamiAlavi** [comment]
Hi @TildaDares, first timer here for this repo.

**Comment by ilaiyengar** [comment]
Hi @TildaDares. First timer here and I would love to contribute!

**Comment by RDiMag** [comment]
Hello! I'd like to contribute. Please add me to the list for an FTO.

**Comment by catmar22** [comment]
Hello @TildaDares  I would love to contribute! Thank you this great opportunity for first timers!

**Comment by TildaDares** [comment]
Hi @SamiAlavi, there's an issue reserved for you here https://github.com/publiclab/plots2/issues/11501.

**Comment by samailamalima** [comment]
Hi @TildaDares I will love to contribute as a FTO any opportunity thanks

**Comment by ishfaque1998** [comment]
Hy @TildaDares  I want to contribute and you are giving  this great opportunity for first timers!

**Comment by meeddhhaa** [comment]
Hey @TildaDares I'm new to open source and I'd love to contribute! Could you please assign me an FTO?

**Comment by TalhaHere12** [comment]
Hey @TildaDares I am new to open source. Could you please assign me an FTO

**Comment by Caldwell-77** [comment]
Hi @TildaDares -- Another first-timer here hoping to contribute. Can you please add me to the queue for an FTO issue? I'd really appreciate the opportunity to work on one when/if possible. Thank you so much!

**Comment by jywarren** [comment]
Hi all! Thank you so much for your interest, we are putting a slight pause
on new FTOs for the time being, but should be able to post more soon.
Thanks for your patience.

**Comment by professorabhay** [comment]
Hello, I am  preparing for GSOC 2023. I want to contribute in  Open Source. I am new here want some guide and solve some GOOD FIRST ISSUES!!
Plz help me out.

**Comment by ayish1998** [comment]
hello I would like to make a contribution since I am preparing for outreach 2023

**Comment by hackerwhale** [comment]
Hello, @TildaDares  I want to contribute to this project and learn about the workflow contribution to FTOs(First Timers Only). Can you please add me to the queue for an FTO issue?

**Comment by FairyPenguin** [comment]
Hello First timer here. I would love to work on an issue.

**Comment by ayish1998** [comment]
@TildaDares I would like to be assigned to a first-timer issue. Thank you

**Comment by ISHA25VERMA** [comment]
> (This issue is a continuation of #10153)
> 
> Hi everyone, we at Public Lab are very welcoming to contributors especially open source newcomers and people from underrepresented groups. Contributing to Open-Source can be overwhelming especially for first-timers and that's why we created this issue to make it easier for you to contribute.
> 
> We have issues that we call FTOs(First Timers Only) that are meant for first-timers only. These issues give you a step-by-step guide on how to contribute and what files need to be changed. You don't have to know any programming language to work on these issues. These issues are designed to help you learn our workflow here at Public Lab.
> 
> ### How Do I Get an FTO Issue?
> Please leave a comment below and we'll add your name to the list. If you're an Outreachy or GSOC applicant, please tell us and we'll give you priority.
> 
> We'll tag you when there's an issue for you to work and you have **24 hours** to claim it. If you're stuck, please feel free to reach out and ask for help in our [community chat](https://gitter.im/publiclab/publiclab).
> 
> ### How To Claim an FTO?
> Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.
> 
> And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.
> 
> If the issue was reserved for you, please claim it **within 24 hours**.
> 
> ### Contributors looking for FTOs
> * [x]  @Favour-Cay
> * [x]  @iLynette
> * [x]  @AM-png
> * [x]  @banjo-tech
> * [x]  @mimi-tech
> * [x]  @onujoy14
> * [x]  @olamideazeez
> * [x]  @Ceejaycodez
> * [x]  @Sally-N
> * [x]  @sharon-kathambi
> * [x]  @victorvictoria-maker
> * [x]  @Kemmie001
> * [x]  @Jolah1
> * [ ]  @M0Hassan
> * [x]  @julianavmgiusti
> * [ ]  @ForJessicaSake
> * [ ]  @Simplecodez
> * [x]  @alexanderchosen
> * [ ]  @thenextxchapter
> * [ ]  @damli40
> * [ ]  @Frank-Cisca
> * [ ]  @Alwoch
> * [x]  @Chelsofia
> * [x]  @davidarebuwa
> * [ ]  @AminMusah
> * [x]  @Chinazaekpere-Ike
> * [ ]  @rsmith-github
> * [ ]  @laskelly
> * [ ]  @Jem256
> * [x]  @Onyi-X
> * [ ]  @RoleyJes
> * [ ]  @hugochooqu
> * [x]  @Marvisjaja
> * [ ]  @hillaryCyn
> * [x]  @yukiyuli
> * [x]  @SylviaKaks
> * [x]  @Natacha-Danbadan
> * [ ]  @aweSAM-XS
> * [x]  @Dhei-vid
> * [ ]  @MARVEBUKA
> * [ ]  @AryaCollection
> * [ ]  @emmyclef
> * [x]  @boluwatifee
> * [ ]  @sanmay321
> * [x]  @lizwe-mac
> * [ ]  @AdityaVerma32
> * [ ]  @TechieTee
> * [ ]  @LynetteMwe
> * [ ]  @faithybest
> * [ ]  @Frank-Cisca
> * [ ]  @EricaP451
> * [ ]  @vareenrose
> * [ ]  @Tonathe3rd
> * [x]  @Dorcas-BD
> * [ ]  @Tishly
> * [x]  @HM-8
> * [ ]  @aditi5050
> * [x]  @Dalu46
> * [ ]  @Augusta2
> * [ ]  @charmilahruthika
> * [ ]  @Wafa9
> * [ ]  @safa-ismail
> * [x]  @Chelsofia
> * [ ]  @RoleyJes
> * [x]  @NadifaHersi
> * [ ]  @samailamalima
> * [ ]  @TEGS-OMAT
> 
> ### I've worked on an FTO, what next?
> Congratulations 🎉, you've made your first Public Lab Contribution. We always encourage contributors to create FTOs for others or look out for issues with the `bug` /`help wanted` tags.
> 
> If you'd like to create an FTO for someone else, you can look through the repo or [publiclab.org](https://publiclab.org/) for issues that can be solved in a few lines. When you've found an issue for a first-timer to work on, click the **New Issue** button and use the issue template named `First Timers Only issue`.
> 
> If you have any trouble, feel free to reach out to us on our [community chat](https://gitter.im/publiclab/publiclab) and we'll do our best to help you resolve it. Thank you 🎈

Hello, I want to be assigned a FTO issue. Planning to participate in gsoc this year.
Thankyou

**Comment by Laraibnehal** [comment]
@TaladeogoA I would like to work in this issue as I'm a first timer and I'm a core member of gdsc at my college in 2nd year

**Comment by italopires** [comment]
I'd like to get a first timer issue, It'd be my first contribution, could someone find one issue and assign to me?

**Comment by vikasjain2003** [comment]
Hello, I am new to open source. Please add me

**Comment by SharoonRafeek** [comment]
Hello, I am a first-timer here, and I would like to contribute. Please add me.

**Comment by Favour-20** [comment]
Hello, I'm new to open source and I would love to join. Please add me

**Comment by xiyini** [comment]
I would like to join!

**Comment by 1san1** [comment]
Hello I am new to open source and like to work on the issue

**Comment by Haoyun-Hong** [comment]
Hello, I am first time contributor and I would like to work on an issue.

**Comment by ilacorda** [comment]
Hi, adding myself to it as well. First time contributor to plots2 and would like to work on potentially any Ruby-related issues. Happy to jump on anything!

**Comment by Javali-M** [comment]
Hello, I am also a first-time contributor to plots2 and would appreciate if an issue is assigned to me. I'm happy to work on anything related to either HTML, CSS or JavaScript.

**Comment by dAnkc0der** [comment]
Hey, I am a first-time contributor and would like to work on an issue.

**Comment by itsgurmannatsohal** [comment]
Hi, I'm contributing to open source for the first time and would like an issue to be assigned to me. I'm efficient in various JavaScript frameworks and c++ but can also learn fairly quickly. 
Thank you

**Comment by MugdhaPadgelwar** [comment]
Hi,I am first-timer contributor and would like to work on an issue.

**Comment by Sreejith2** [comment]
Hi iam new to opensource i would like to work on an issue please add me

**Comment by vishvampawar11** [comment]
Hello, I am a first time contributor and I would like to work on the issue.

**Comment by jywarren** [comment]
Hello folks, we are a bit slow these days but will try to find some good
issues for you soon. I do want to let you know that we will not be
participating in GSOC this year. Thank you for your interest!!

**Comment by ifeanyiHM** [comment]
Hi @TildaDares I'm a first timer and a GSOC prospect. I would like to work on an issue. Thank you very much

**Comment by Teejay128** [comment]
Hi there, I am also a beginner to open source and would like to work on a issue.

Thank you very much

Also just learned you will not be participating in this year's GSoC, 😞

**Comment by aishwarya-mali** [comment]
Hi @TildaDares, I'm a first time contributor to this repository and would love to work on a issue. Thank you!

**Comment by palash018** [comment]
Hi @TildaDares. I'm a first time contributor,I really want to contribute as I have gained some experience in react and nodejs

**Comment by mithindev** [comment]
Hello, I am new to the open source community and I am eager to contribute by working on an issue. I would be grateful for any opportunities to get involved. Thank you for your consideration.

**Comment by Blueclaus13** [comment]
Hello!! I'm first time contributor, I would like to work on an issue, thanks

**Comment by Medha2608** [comment]
Hello, I'm new to open source and would like to work on an issue, thanks!

**Comment by benluiwj** [comment]
hello! I am new to open source and would like to work on an issue, thanks 😄

**Comment by HimanshuMathur1396** [comment]
Hello I am new to open source and would like to work on an issue. Thanks

**Comment by 120EE0980** [comment]
Heloo..I want to work on an issue.

**Comment by ayish1998** [comment]
Hello, I am Outreachy applicant and would like to contribute to an open source. I have knowledge in JavaScript, Flutter.

**Comment by Akhilbisht798** [comment]
Hey First timer here like to contribute to this project

**Comment by FrancoFrancis** [comment]
Hi, Would love to be assigned an issue

**Comment by emilianolch** [comment]
Hello, I'm a first timer. I'd like to contribute to this project.

**Comment by hound77** [comment]
@TildaDares I want to contribute, I am first timer and new to open source.

**Comment by Princedoshi** [comment]
Hello, I'm a first timer. I'd like to contribute to this project.

**Comment by qasim006** [comment]
Hi, I would like to help!

**Comment by jonhoa** [comment]
First timer here. Would love to contribute to a project that runs on React, Ruby, HTML, CSS or Javascript.

**Comment by saksham28hans** [comment]
Hey, I am new to Open Source. I would like to contribute.

**Comment by antonypauson** [comment]
hey I would love to contribute to something here.

**Comment by jywarren** [comment]
Hi everyone, I'm sorry to say we are taking a pause due to limited resources and capacity and are hoping to start up again in about 6 months. I'll definitely check back in here once we are ready to start up again!

**Comment by arunraj3** [comment]
Hi everyone , I'm New to open source can i work on this issue?

**Comment by Jayyk09** [comment]
I'd like to be added to the list! Thanks

**Comment by verz0** [comment]
Hey, I'm new to open souce and I'd like to conrtibute to a FTO issue!

**Comment by rishabhpro** [comment]
Hi! I'm a first timer and I'd love to work on something!

**Comment by yashp4858** [comment]
@TildaDares Hi first timer I would like to work on issues

**Comment by aditya123473892** [comment]
hi i am a first timer want to work on a issue

**Comment by saksham-tomer** [comment]
hey @TildaDares  i am a first timer and i am looking forward to work on an issue

**Comment by owhynotme** [comment]
Hey, @TildaDares I'd like to contribute to a FTO issue!

**Comment by roryai** [comment]
I'd like to work on this project, please sign me up or add me to an issue. I have a few years Rails experience.

**Comment by frodri13** [comment]
Hello there, commenting here to contribute to a FTO. Please sign me up

**Comment by nayem-mehedi** [comment]
Hello, first timer here. I will be happy to contribute.

**Comment by areejnaqvi** [comment]
hello, first timer here. I would love to be able to contribute! :)

**Comment by bryankhor07** [comment]
Hello First timer here. I would love to work on an issue.

**Comment by ramya202000** [comment]
Hello @TildaDares I am a first timer here. I would like to contribute.

**Comment by daluclemas** [comment]
hi @TildaDares i would like to contribute to any project. Please help.

**Comment by chaitali-khangar** [comment]
Hi @TildaDares I am a first timer. I would love to contribute.

**Comment by joshirakesh** [comment]
Hi @TildaDares, I am a first-time contributor and am very interested in helping out with this project. Looking forward to contributing!

**Comment by NehaL-8** [comment]
Hii First timer here! Intreseted in contibuting to your project.

**Comment by naseehasalam** [comment]
Hello ! @TildaDares , This is my first time experiencing the open source world. I have been doing ruby for a while now and was hoping to try for GSoc in 2024. I was hoping you could help me find my first-timer-only issue. I have opened a pull request on one which fixes #11620 .

**Comment by NehaL-8** [comment]
Hello! This is my first time trying to make pull requests ,it would be
great to have the right guidance to move further and achieve it.

**Comment by naseehasalam** [comment]
Hey!, sure I would love to help you out even though I am a beginner myself.
Can you tell me exactly where you are finding trouble?

**Comment by harshsinghcs** [comment]
Hello ! @TildaDares  This is my first time experiencing the open-source world. I have been doing Java script for a while now and was hoping to try for GSoc in 2024. I was hoping you could help me find my first-timer-only issue. 
@TildaDares

**Comment by Mrinalinini** [comment]
Hey there @TildaDares ! I am a first timer student and it would be very needful if you give me an issue to solve. Since this includes in my project, I would be very pleased if you give me this opportunity. Please help me with this. Thankyou !

**Comment by Saumya69** [comment]
Hello, @TildaDares I hope you're well!!This is first time contributing to an open source project.I have been using python ,html,css ,SQL for a while now.I was hoping you could find me a first timers issue to get started on.This will also help me with my college projects.Thank you

**Comment by noelsebastian22** [comment]
Hi @TildaDares ,

I hope this message finds you in good spirits. As an experienced Angular/JavaScript developer, I'm thrilled to take part in my first open-source contribution through your project. My strong command of HTML, CSS, and SQL adds to my skill set. To kickstart this journey, could you suggest a suitable first-timers issue? Your guidance will be invaluable as I navigate this new endeavor. Thank you for the opportunity!

**Comment by Shrey-Raj** [comment]
Hi there ! I am a first time contributor .
Looking forward to contributing to PublicLab for the first time! 
Excited to be part of this community and make a difference .

**Comment by Cyb3r-punk6969** [comment]
Hello there,i am new to open source contribution.Would love to help.

**Comment by ifeawoyemi** [comment]
Hi @TildaDares please I am new here, i would love to contribute, can you assign me FTO

**Comment by ghost** [comment]
Hi there! first timer here.
Want to contribute

**Comment by srihari20031** [comment]
Hi There! A First timer here, Would love to contribute, Kindly assign me some issues to solve.!

**Comment by rwood219** [comment]
Hello, looking for some FTO issues. Please add me to the list. Thank you.

**Comment by Dalu46** [comment]
Hi @TildaDares I'm a first timer and I'd love to work on an FTO

**Comment by noemielach** [comment]
Hi I'm first timer and I would like to be added to the list please :)

**Comment by pijusmulevicius** [comment]
Hi, it's my first time with open source. I would like an FTO please

**Comment by knotinline** [comment]
Hi, I'd love to be added to the list :)

**Comment by Ayush-2001-Dhanraj** [comment]
Hi! @TildaDares  I'm a first timer and I'd love to work on something!

**Comment by anitacheung83** [comment]
Hi! @TildaDares, I am a first timer and I'd love to be added on the list! Thank you :)

**Comment by Marukos** [comment]
Hello! I am a first timer and I would love to learn how to contribute to the open source community by starting with an issue of yours!

**Comment by SDR01** [comment]
Hello @TildaDares, first-timer here excited to contribute.

**Comment by LakshmiGayathri19** [comment]
Hello @TildaDares , 

I would like to work on an issue and this is my first time. I am excited to contribute to some challenging issues. Could you please assign me to any interesting issue ?

Thanks in advance.

**Comment by RenataJosipovic** [comment]
Hi @TildaDares,
I am a first timer and I would like to contribute by working on an issue.

**Comment by adesh1998** [comment]
Hello @TildaDares , I would like to work on some issue could you please assign me one?

**Comment by yuzanawinnnn** [comment]
Hi @TildaDares, I am complete beginner in opensource contribution and learning to have a great understanding of how things work on Github. I would like to start contributing by working on an issue. Could you assign me one? Thank you

**Comment by shehryarusman** [comment]
Hi @TildaDares! I am new to the open source world and would really like to get started somewhere to hopefully a long journey of open source contributions! Please let me know if there are any issues I can be assigned to! Thank you!

**Comment by abdul-rehman-2050** [comment]
Hi, I would like to be tagged to FTO Thanks

**Comment by kkartik07** [comment]
Hello @TildaDares I am new here and looking forward to contribute as well. Please add me to the FTO list.

**Comment by jywarren** [comment]
Hello, unfortunately we aren't able to participate in GSoC, thank you for
your interest!
